<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 241);

font: 63 10pt &quot;Cascadia Code SemiBold&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>511</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>480</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 237, 193);
</string>
    </property>
    <property name="text">
     <string>21132_Shreyash</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 237, 193);</string>
    </property>
    <property name="text">
     <string>Cohen-Southerland line clipping algorithm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>161</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="DrawLines">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 237, 193);</string>
       </property>
       <property name="text">
        <string>Draw Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DrawWindow">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 237, 193);</string>
       </property>
       <property name="text">
        <string>Draw Window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClipButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 237, 193);</string>
       </property>
       <property name="text">
        <string>Clip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SelectColor">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 237, 193);</string>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 237, 193);</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAssignment_5">
    <property name="title">
     <string>Assignment 5</string>
    </property>
   </widget>
   <addaction name="menuAssignment_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
